bigtable/src/main/scala/magnolify/bigtable/BigtableType.scala

```diff
@@ -127,8 +148,24 @@ object BigtableField {
     private def key(prefix: String, label: String): String =
       if (prefix == null) label else s"$prefix.$label"
 
-    override def get(xs: java.util.List[Column], k: String): T =
-      caseClass.construct(p => p.typeclass.get(xs, key(k, p.label)))
+    override def get(xs: java.util.List[Column], k: String): Result[T] = {
+      var fallback = true
```

Comment:
I m wondering if this could be implemented without `var`, but probably this isn't going to be a simple change. I checked `magnoliy` and not seeing any stateful `construct` equivalents.

Reply:
`caseClass.rawConstruct(fieldValues: Seq[Any])` so we can technically do a `caseClass.parameters.foldLeft()` with `z: (Seq[Any], Boolean)` but that feels less readable :)

Plus this state is never leaked and shouldn't cause threading issue.

Reply:
Yeah, I had an idea to fold over parameter, but didn't know there is a `rawConstruct` function. good to know! I agree, it would be over-complication w/o any significant benefits.